jwst/cube_build/ifu_cube.py

```diff
@@ -2291,6 +2291,13 @@ def setup_final_ifucube_model(self, model_ref):
             ifucube_model.meta.ifu.roi_wave = 0
             ifucube_model.meta.ifu.roi_spatial = 0
 
+            # uncorrect cdelt for degree conversion
+            ifucube_model.meta.wcsinfo.cdelt1 *= 3600.0
+            ifucube_model.meta.wcsinfo.cdelt2 *= 3600.0
+
+            # correct "RA" axis orientation
+            ifucube_model.meta.wcsinfo.pc1_1 *= -1.0
```

Comment:
Good catch. A lot of folks forget about the fact that RA runs right to left on the sky!